cime_config/machines/config_compilers.xml

```diff
@@ -914,6 +918,52 @@ flags should be captured within MPAS CMake files.
   <PNETCDF_PATH>$ENV{PNETCDF_PATH}</PNETCDF_PATH>
 </compiler>
 
+<compiler MACH="chrysalis" COMPILER="intel">
+  <CPPDEFS>
+    <append COMP_NAME="gptl">-DHAVE_SLASHPROC</append>
+  </CPPDEFS>
+  <MOAB_PATH>/lcrc/soft/climate/moab/chrysalis/intel</MOAB_PATH>
+  <FFLAGS>
+    <append DEBUG="FALSE">-O2 -debug minimal -qno-opt-dynamic-align</append>
+  </FFLAGS>
+  <PIO_FILESYSTEM_HINTS>gpfs</PIO_FILESYSTEM_HINTS>
+  <SLIBS>
+    <append>$SHELL{$ENV{NETCDF_FORTRAN_PATH}/bin/nf-config --flibs} $SHELL{$ENV{NETCDF_C_PATH}/bin/nc-config --libs} -mkl</append>
+  </SLIBS>
+  <NETCDF_C_PATH>$ENV{NETCDF_C_PATH}</NETCDF_C_PATH>
```

Comment:
@iulian787 Need to add MOAB install location and flags etc for Compy and Cori as well.